app/assets/stylesheets/bali/variables.scss

```diff
@@ -0,0 +1,23 @@
+// box-shadow
+$box-shadow: 0px 3px 18px rgba($black, 0.1), 0 0 0 1px rgba($black, 0.03);
+
+// colors
+$white: hsl(0, 0%, 100%);
+
+// font-weight
+$fw-500: 500;
+
+// height
+$navbar-height: 3.75rem;
+
+// sizes
+$size-6: 1rem;
+$size-7: 0.875rem;
+
+// width
+$desktop-side-menu-width: 12rem;
+$side-menu-width: 14rem;
```

Comment:
This should go inside the side-menu CSS

Reply:
`$navbar-height` too

Reply:
and the `$z-index-side-menu:` ..

Reply:
I think `$navbar-height` should stay in this file because it is a reference to how far down the html elements should be so that it doesn't cover the navbar